core/app/models/spree/tax/output/order.rb

```diff
@@ -0,0 +1,10 @@
+module Spree
+  module Tax
+    module Output
+      class Order
+        include ActiveModel::Model
+        attr_accessor :id, :line_items, :shipments
```

Comment:
The current implementation allows for order-level tax adjustments. For all sane configurations I know we shouldn't allow that, just noting that this is conspicuously absent here.